src/utils/get-import-flavor-of-node.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    importFlavorIgnore,
+    importFlavorRegular,
+    importFlavorSideEffect,
+    importFlavorType,
+} from '../constants';
+import type { GetImportFlavorOfNode } from '../types';
+
+/**
+ * Classifies nodes by import-flavor, primarily informing whether the node is a candidate for merging
+ *
+ * @param node
+ * @returns {("prettier-ignore"|"regular"|"side-effect"|"type")}
+ */
+export const getImportFlavorOfNode: GetImportFlavorOfNode = (node) => {
```

Comment:
It seems like some of this is duplicated logic with `getChunkTypeOfNode`, perhaps it can be consolidated?